onnxruntime/core/providers/cpu/tensor/scatter_nd.cc

```diff
@@ -11,73 +12,87 @@ ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
     11,
     12,
     KernelDefBuilder()
-        .TypeConstraint("T", DataTypeImpl::AllTensorTypes())
-        .TypeConstraint("Tind", DataTypeImpl::GetTensorType<int64_t>()),
```

Comment:
Clean up. There is no real Tind as per the spec. The indices tensor is always int64_t type.